recipe/meta.yaml

```diff
@@ -11,18 +11,18 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 0
-  skip: true  # [win]
+  number: 1
+  noarch: generic
   script:
-    - make install prefix=$PREFIX
+    - make install prefix=$PREFIX 
```

Comment:
It looks like this change just adds a space at the end of the line ... can you take that out?